controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -145,7 +145,7 @@ module.exports = async (req, res) => {
                                 a.is_anonymous = false
                                 AND a.is_banned = false
                                 AND a.blocked_by_admin = false
-                                AND a.karma_score > 30
+                                AND a.karma_score > 45
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the increase in the karma score threshold may affect the users returned by this query. Confirm that this is the intended behavior.
```diff
- AND a.karma_score > 30
+ AND a.karma_score > 45
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->